src/plugins/intel_gpu/src/plugin/ops/constant.cpp

```diff
@@ -109,7 +109,8 @@ static void CreateConstantOp(Program& p, const std::shared_ptr<ngraph::op::v0::C
         } else if (ngraph::op::is_binary_elementwise_arithmetic(outOp) ||
                    ngraph::op::is_binary_elementwise_logical(outOp) ||
                    ngraph::op::is_binary_elementwise_comparison(outOp) ||
-                   ngraph::is_type<ngraph::op::v0::SquaredDifference>(outOp)) {
+                   ngraph::is_type<ngraph::op::v0::SquaredDifference>(outOp) ||
+                   ngraph::is_type<ngraph::op::v0::Convert>(outOp)) {
```

Comment:
This is applied too broad range of cases...

Instead, I suggest to do like, 
 if "outOp" is convert, than check whetherits predecessor is one of "binary_elementwise_arithmetic, binary_elementwise_logical, binary_elementwise_comparison, squareddifference" and then do the original logic